ci/jenkins/jobs/projects-lab.yaml

```diff
@@ -549,6 +549,62 @@
           started_status: null
           wrappers: []
           publishers: []
+      - '{name}-{test_name}-for-pull-request':
+          test_name: windows-conformance
+          node: 'antrea-windows-testbed'
+          description: 'This is the {test_name} test for {name}.'
+          branches:
+          - ${{sha1}}
+          builders:
+            - builder-conformance-win:
+                conformance_type: 'windows-conformance'
```

Comment:
Without adding `contaienrd` in type value, this test cannot be built based on the correct snapshot in test.sh.